source/Calamari.Common/Variables/JavaVariables.cs

```diff
@@ -0,0 +1,7 @@
+namespace Calamari.Common.Variables
+{
+    public static class JavaVariables
```

Comment:
This is only ever used by the Java steps. Should it be in `common`?

Reply:
It's a bit of a foreshadow of what will be moving with the rest of Common. Package extraction is moved into common which includes Jars. I could remove this now until we merge that stuff into common too?

Reply:
I would have thought that anything that is not needed in the new world would stay in `Calamari.Shared`, or be merged back into `Calamari`. The Java steps and Maven extraction would not be needed by `Calamari.Terraform` or AWS/Azure steps. The only thing that sets this variable is whatever uses the `JavaTools`